scripts/jobs/validate/publish-core

```diff
@@ -28,7 +28,7 @@ case $prDryRun in
     if $libraryAvailable && $reflectAvailable && $compilerAvailable; then
       echo "Scala core already built!"
     else
-      $SBT -warn "setupPublishCore $prRepoUrl" publish
+      $SBT -Dscala.build.publishDevelocity=build -warn "setupPublishCore $prRepoUrl" publish
```

Comment:
Why not using the name of the script: `publish-core` instead of `build`?

Reply:
The value is turned into the `JENKINS_STAGE` value on build scans, eg https://develocity.scala-lang.org/s/gqtejqozpexas/custom-values, `build` and `test` just looked nice 🤷‍♂️